components/titiler/sidebar.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import {
+  Sidebar as SidebarComponent,
+  SidebarDivider,
+} from '@carbonplan/layouts'
+import { Box, Flex } from 'theme-ui'
+
+export default function Sidebar() {
```

Comment:
>I assumed that carbonplan/map example and titiler example would live on separate pages. but maybe we want them to live on the same page so users can toggle between them?

I do think ideally we will able to toggle between them (see the early mock below), but I think defining them on separate pages could be completely reasonable way to do that. Would it be possible to reuse the existing `components/sidebar.js` for the titiler example? If we do that, then we should be able to add an approach toggle to the sidebar, wire up the sidebar properties to routes, and ingest the route parameters for both approaches.

Does that plan sound reasonable? Also totally fine with getting the titiler example up and running in the app first before making the changes proposed above if that'd be easier.

![image](https://github.com/carbonplan/prototype-maps/assets/12436887/e577d799-8a6b-4e27-bc40-f9d47048268f)

Reply:
@hanbyul-here I took a stab at this sidebar refactor + the routing update here: https://github.com/carbonplan/prototype-maps/pull/9. Let me know how this looks to you / if you'd like to talk through the integration steps!